### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by AlessandroBono** [bmwcarit/python-dlt#1]
Hello, is it possible to adapt this library to work on Windows too?

**Comment by aigarius** [comment]
We do not use it on Windows internally and we have no experience in making the ctypes work on Windows. If someone can make it work and the patch does not look too horrible, we can include that functionality. At least the part that loads the C library (.so in Linux and .dll in Windows) likely needs to change. The rest *should* be platform agnostic, but you never know unless you actually try.

**Comment by aigarius** [comment]
Yes, the library comes from https://github.com/GENIVI/dlt-daemon there are some references to it working in Windows at https://at.projects.genivi.org/wiki/display/PROJ/About+DLT-Transport

**Comment by ankitkgupta09** [comment]
Did anyone do modification in library in order to work on Windows as well?

**Comment by Kochise** [comment]
Hi, about the 'ctypes' issue, maybe you would try to consider 'pywin32-ctypes' :

https://github.com/enthought/pywin32-ctypes

As for installation, would a "pip" package be available, like :

pip install python-dlt

Thanks...

**Comment by KheireddineAzzez** [comment]
is this library is mounted on pip, so we can installed

**Comment by luxoft-yuriy** [comment]
Hello colleagues seems publicly available in [python-dlt](https://pypi.org/project/python-dlt/) is not bmw/python-dlt /* follow the link */ , it is a [data load tool (dlt)](https://github.com/dlt-hub/dlt)
